src/algebra/lie_algebra.lean

```diff
@@ -174,23 +180,26 @@ end prio
 
 namespace lie_algebra
 
-variables (R : Type u) (L : Type v) [comm_ring R] [add_comm_group L] [lie_algebra R L]
-
 /--
-The adjoint action of a Lie algebra on itself.
+A morphism of Lie algebras is a linear map respecting the bracket operations.
 -/
-def Ad (x : L) : L →ₗ[R] L :=
-{ to_fun := has_bracket.bracket x,
-  add    := by { intros, apply lie_add },
-  smul   := by { intros, apply lie_smul } }
+structure morphism (R : Type u) (L : Type v) (L' : Type v)
+  [comm_ring R] [add_comm_group L] [lie_algebra R L] [add_comm_group L'] [lie_algebra R L']
+  extends linear_map R L L' :=
+(bracket : ∀ {x y : L}, to_fun ⁅x, y⁆ = ⁅to_fun x, to_fun y⁆)
```

Comment:
```suggestion
(map_lie : ∀ {x y : L}, to_fun ⁅x, y⁆ = ⁅to_fun x, to_fun y⁆)
```
Sorry for coming back to these silly issue of names. But I think `map_lie` is more in line with `map_add` and friends. As such, this is the name that users would expect, I guess. Please take a look at the other names, to see if they need updating as well.

Reply:
It's not silly at all; as a consumer of mathlib I have some appreciation of the value of the good naming that is already in place. I really do try to come up with sensible names, even though that must seem doubtful at times!